index.js

```diff
@@ -1,21 +1,26 @@
-
-
-function submitbtn() {
-    document.querySelector('.secondFace').style.display = 'block';
-    var getSelectedValue = document.querySelector(
-        'input[name="score"]:checked');
-
-    if (getSelectedValue != null) {
-        document.getElementById("selected").innerHTML
-            = getSelectedValue.value;
+/**
+ * Shows in DOM the secondFace section, along with the selected value.
+ * @param {HTMLInputElement} selectedValue
+ */
+const renderSecondFade = (selectedValue) => {
+    // Looks for elements in DOM before changing them.
+    const secondFace = document.querySelector('.secondFace');
+    const selectedSpanText = document.querySelector("#selected");
+    // Checks that the secondFace and selectedSpanText exits in DOM.
+    if (secondFace && selectedSpanText) {
+        secondFace.style.display = 'block';
+        selectedSpanText.textContent = selectedValue.value;
     }
+}
 
-    var getSelectedValue = document.getElementById("score-btn").value;
-
-    if (getSelectedValue != null) {
-        document.getElementById("selected").innerHTML
-            = getSelectedValue.value;
-
+/**
+ * Looks for the selected value,
+ * then triggers the secondFace section just if the user selects a value.
+ */
+const handleSubmit = () => {
+    const getSelectedValue = document.querySelector(
+        'input[name="score"]:checked');
+    if (getSelectedValue) {
+        renderSecondFade(getSelectedValue)
```

Comment:
Function separated into small functions to improve code readability and maintainability.